minio/minio
# Adds info on policy for STS authentication using web-id

Fix for #9248

This change adds more information on minimum required policy to have a reasonable test outcome to verify sts web-id functionality works.

## Motivation and Context
Issue#9248
User thinks there is not enough documentation about the policy required for sts authorization with web-id. 


## How to test this PR?
Assign the policy provided in `web-identity.md` doc to a user and check if the policy resrtictions and allowed actions are followed by when user attempts to run different kinds of commands.

## Types of changes

## Checklist:
